tests/wpt/metadata/html/semantics/forms/the-input-element/range-2.html.ini

```diff
@@ -9,6 +9,6 @@
   [range input value equals 100]
     expected: FAIL
 
-  [range input value equals 2]
+  [range input value set to an integer]
```

Comment:
Why does this test now fail?

Reply:
Originly pass because of accident, in other words, it is lucky.
Origin logic doesn't do anything, but we now add logic.

Reply:
Can you elaborate on what this accident is about? Is it a case of where two negatives makes a positive?

Reply:
there is no function to check the value originally, so the output is still as same as input. But the new code do some checking, and without `step` inside, the checking is not completed. That's why here is a test become to failed.